flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -71,6 +72,8 @@
  */
 public class FlinkCatalog extends AbstractCatalog {
 
+  private final CatalogLoader catalogLoader;
+  private final Configuration hadoopConf;
```

Comment:
Those two seems don't have to be private members because I did not see anywhere accessing them except the constructor.